### ISSUE TITLE
Regarding Results 

### ISSUE THREAD

**Issue by himashi92** [HuCaoFighting/Swin-Unet#26]
Hello authors,

You have done a fabulous job in implementing this approach. Thanks for making the repo publicly available.

I tried to train swin-unet with Synapse data(got it from transunet authors) but the results are not around the results you mentioned in the paper. I got mean dice of 0.780137 & mean hd95 of 30.579858.

I used the following hyper params according to your ReadME doc and config file.

--max_epochs 150 --img_size 224 --base_lr 0.05 --batch_size 24
    EMBED_DIM: 96
    DEPTHS: [ 2, 2, 2, 2 ] (This is quiet contradictive. In the paper it is  [2, 2, 2, 1])
    DECODER_DEPTHS: [ 2, 2, 2, 1]
    NUM_HEADS: [ 3, 6, 12, 24 ]
    WINDOW_SIZE: 7

Appreciate if you can provide the hyper params that you have used to get the results mentioned in the paper.

Thanks,
Himashi

**Comment by zhouyi-git** [comment]
Can you get the parameters? I also ran to 0.78

**Comment by himashi92** [comment]
@zhouyi-git No, I didn't. But I tried depth of [2 2 2 1] and got values around 0.79

**Comment by zhouyi-git** [comment]
Thanks




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "HuCaoFighting/Swin-Unet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月21日(星期二) 上午8:02
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [HuCaoFighting/Swin-Unet] Regarding Results  (#26)





 
@zhouyi-git No, I didn't. But I tried depth of [2 2 2 1] and got values around 0.79
  
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by zhenxing123** [comment]
> 你好作者，
> 
> 您在实施这种方法方面做得非常出色。感谢您将回购公开。
> 
> 我尝试使用 Synapse 数据（从 transunet 作者获得）训练 swin-unet，但结果与您在论文中提到的结果不同。我的平均骰子为 0.780137，平均 hd95 为 30.579858。
> 
> 我根据您的自述文件和配置文件使用了以下超参数。
> 
> --max_epochs 150 --img_size 224 --base_lr 0.05 --batch_size 24 EMBED_DIM: 96 DEPTHS: [ 2, 2, 2, 2 ] （这很矛盾。在论文中它是 [2, 2, 2, 1] ) DECODER_DEPTHS: [ 2, 2, 2, 1] NUM_HEADS: [ 3, 6, 12, 24] WINDOW_SIZE: 7
> 
> 感谢您是否可以提供用于获得论文中提到的结果的超参数。
> 
> 谢谢， 喜马仕



> @zhouyi-git不，我没有。但是我尝试了 [2 2 2 1] 的深度并得到了大约 0.7

**Comment by zhenxing123** [comment]
你好，可以分享一下作者预处理数据集吗？方便的话可以发邮箱，zan1062829630@gmail.com

**Comment by wangru1026** [comment]
有没跑通了，可以交流一下吗？我现在不知道是数据集问题还是模型加载问题，测试结果为0是什么鬼？

**Comment by zhouyi-git** [comment]
跑过了 按照他那个要求搞就是




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "HuCaoFighting/Swin-Unet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月25日(星期天) 晚上10:35
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [HuCaoFighting/Swin-Unet] Regarding Results  (#26)





 
有没跑通了，可以交流一下吗？我现在不知道是数据集问题还是模型加载问题，测试结果为0是什么鬼？

**Comment by HuCaoFighting** [comment]
您好，您的邮件已收到。

**Comment by YumnaMemon** [comment]
> When I tested, I never reached the author's accuracy. Except for batch_size, which was changed to 12, the other parameters were not changed. I don't know where the problem lies. Can anyone help me solve it?

have you solve it, connect with me on wechat: yahm30

**Comment by syhkqa** [comment]
我测试时一直没有达到作者的精度，除了batch_size改成了12，其他参数都没有动，不知道问题出在了哪，有没有大佬帮忙解答一下？

我也是，你解决了吗

**Comment by syhkqa** [comment]
> 我测试时一直没有达到作者的精度，除了batch_size改成了12，其他参数都没有动，不知道问题出在了哪，有没有大佬帮忙解答一下？

请问你解决了吗
